src/Nether.Analytics/IMessageRouter.cs

```diff
@@ -7,7 +7,7 @@ namespace Nether.Analytics
 {
     public interface IMessageRouter
     {
-        //TODO: Make method Async
-        Task RouteMessageAsync(Message msg);
+        Task RouteMessageAsync(string partitionId, Message msg);
```

Comment:
he additional parameter partitionId is there since I sometimes need to run things without having a message, but I still need the partitionId. I'm considering removing the partition from the messageId instead in a later PR

Also, all messages coming in on one thread will have the same partitionId, so there can be some optimizations by not looking into each and every message, just to find out that they are all the same